qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ControlledActorClockIT.java

```diff
@@ -41,7 +43,9 @@
 
 @Testcontainers
 public class ControlledActorClockIT {
```

Comment:
:x: Oh, one small thing, this sounds to me like we're testing the `ControlledActorClock`, not the actuator endpoint, so maybe we can rename this to `ControlledActorClockEndpointIT`?